hedera-node/src/main/java/com/hedera/services/txns/token/TokenWipeTransitionLogic.java

```diff
@@ -51,24 +59,52 @@
 
 	private final TokenStore store;
 	private final TransactionContext txnCtx;
+	private final TypedTokenStore tokenStore;
+	private final AccountStore accountStore;
+	private final OptionValidator validator;
 
 	public TokenWipeTransitionLogic(
+			final OptionValidator validator,
 			TokenStore store,
+			TypedTokenStore tokenStore,
+			AccountStore accountStore,
 			TransactionContext txnCtx
 	) {
 		this.store = store;
 		this.txnCtx = txnCtx;
+		this.tokenStore = tokenStore;
+		this.accountStore = accountStore;
+		this.validator = validator;
 	}
 
 	@Override
 	public void doStateTransition() {
 		try {
 			var op = txnCtx.accessor().getTxn().getTokenWipe();
```

Comment:
nit: Can we add the inline docs for the different steps we have (parse grpc, load model etc)